src/test/ValidatorKeys_test.cpp

```diff
@@ -136,60 +135,121 @@ class ValidatorKeys_test : public beast::unit_test::suite
             auto const keyType = KeyType::ed25519;
```

Comment:
Good observation. At one point, I was considering omitting the `"key_type"` field and determining it from the public key (`publicKeyType()`). I'm going to take a closer look at that option.